WooCommerce/Classes/POS/Controllers/PointOfSaleCouponsController.swift

```diff
@@ -64,20 +66,47 @@ import protocol Yosemite.PointOfSaleCouponServiceProtocol
 
 @available(iOS 17.0, *)
 private extension PointOfSaleCouponsController {
+    /// Loads the first page by attempting to load the first page from local storage
+    /// then syncs from the remote regardless the result
+    func loadFirstPage() async {
+        do {
+            let storedCoupons = try await couponProvider.provideLocalPointOfSaleCoupons()
+
+            if !storedCoupons.isEmpty {
+                setCouponsLoadedViewState(storedCoupons, hasMoreItems: true)
+            }
+            let coupons = try await couponProvider.providePointOfSaleCoupons(pageNumber: 1)
```

Comment:
We have `Constants.firstPage` in the file but I think using number `1` is even better. So we can remove that constant 😀

Reply:
That's true haha, removed https://github.com/woocommerce/woocommerce-ios/pull/15499/commits/43d955e3e7b23d1f5c8e1924fee6d7ec77bc3d68